packages/cyberstorm/src/components/Markdown/Markdown.module.css

```diff
@@ -0,0 +1,145 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  color: var(--color-text--default);
+}
+
+/*
+  TODO: Use proper styles instead of these placeholder values. The
+  content rendered on this page should match how the content is actually
+  rendered on the package page. I don't think Figma contains all the
+  stuff supported by markdown, so this needs to be discussed further and
+  all the styles should be updated all at once.
+
+  We can't control the elements inside the Markdown wrapper, e.g. to
+  give them class names, and linting requires pure selectors, so repeat
+  the .root class name on each selector.
+*/
+.root h1,
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6 {
+  display: flex;
+  gap: 0.5rem;
+  align-items: flex-start;
+  align-self: stretch;
+  padding: 2rem 0 0.5rem;
+  border-bottom: 1px solid #29295b;
```

Comment:
Please use color variables. Applies to rest of the hex colors used as well.

Reply:
Later alligator